gym_ai2thor/config_files/rainbow_example.json

```diff
@@ -17,6 +17,7 @@
     "resolution": [64, 64],
     "task": {
         "task_name": "PickUp",
-        "target_objects": {"Mug": 10}
+        "target_objects": {"Mug": 10},
+        "movement_reward": -1
```

Comment:
again, movement_reward == -1 is crazy. Why not -0.1 or -0.01
You need to pick up 500 cups (optimal max since pickup and putdown) to get an optimal score of -500?